### PR TITLE
Add support for MCC/MNC and Country fields in CCT Transport Backend.

### PR DESCRIPTION
**Pull request by rlazo**

### CODE REVIEW

File: transport/transport-backend-cct/src/main/java/com/google/android/datatransport/cct/CctTransportBackend.java [hunk 1]
```diff
@@ -136,6 +146,8 @@ public EventInternal decorate(EventInternal eventInternal) {
         .addMetadata(KEY_TIMEZONE_OFFSET, getTzOffset())
         .addMetadata(KEY_NETWORK_TYPE, getNetTypeValue(networkInfo))
         .addMetadata(KEY_MOBILE_SUBTYPE, getNetSubtypeValue(networkInfo))
+        .addMetadata(KEY_COUNTRY, Locale.getDefault().getCountry())
```
Comment (vkryachko) [inline transport/transport-backend-cct/src/main/java/com/google/android/datatransport/cct/CctTransportBackend.java:149]: Just noticed that while we are at it we can also add `Locale#getLanguage()`, do you mind adding it as well?

**Review by vkryachko — COMMENTED** [review]
Can you pls add some tests for this new behavior?

**Review by jeremyjiang-dev — APPROVED** [review]
Thank you Rodrigo for such a quick support! The code and results that you posted in the tickets LGTM.

**Comment by rlazo** [comment]
@vkryachko tests added, PTAL

**Review by vkryachko — APPROVED** [review]
Cool thanks

**Pull request merged**